src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -325,6 +325,242 @@ private AdsDataProvider() {
+                "\"wpage\":\"dark_magician\"",
+                "\"ref\":\"direct\"",
+                "\"lang\":\"en\"",
+                "\"pv\":\"1\"",
+                "\"top\":\"1k\"",
+                "\"esrb\":[\"everyone\"]",
+                "\"age\":[\"teen\",\"kid\",\"18-34\"]",
+                "\"gnre\":[\"anime\",\"fantasy\",\"strategy\",\"comics\"]",
+                "\"media\":[\"tv\",\"cards\"]",
+                "\"pform\":[\"pc\",\"psp\"]",
+                "\"sex\":[\"m\"]"
+            ),
+            Arrays.asList(
+                "\"loc\":\"top\"",
+                "\"pos\":\"TOP_LEADERBOARD\"",
+                "\"src\":\"gpt\""
+            )
+        },
+        {
+            "fallout",
+            "Fallout",
+            "wka.gaming/_fallout//article",
+            "TOP_RIGHT_BOXAD",
+            Arrays.asList(
+                "\"s0\":\"gaming\"",
+                "\"s0v\":\"games\"",
+                "\"s0c\":[\"gaming\"]",
+                "\"s1\":\"_fallout\"",
+                "\"s2\":\"article\"",
+                "\"ar\":\"4:3\"",
+                "\"artid\":\"948\"",
+                "\"cat\":[\"fallout\"]",
+                "\"dmn\":\"wikiacom\"",
+                "\"hostpre\":\"fallout\"",
+                "\"lang\":\"en\"",
+                "\"wpage\":\"fallout\"",
+                "\"ref\":\"direct\"",
+                "\"pv\":\"1\"",
+                "\"top\":\"1k\"",
+                "\"sex\":[\"m\"]",
+                "\"age\":[\"under18\",\"18-34\",\"18-24\",\"25-34\",\"teen\"]",
+                "\"gnre\":[\"action\",\"adventure\",\"fps\",\"openworld\",\"rpg\",\"scifi\"]",
+                "\"pform\":[\"pc\",\"xbox360\",\"ps3\"]",
+                "\"pub\":[\"bethesda\",\"bethesda\"]",
+                "\"esrb\":[\"mature\"]",
+                "\"theme\":[\"military\",\"postapocalypse\"]"
+            ),
+            Arrays.asList(
+                "\"loc\":\"top\"",
+                "\"pos\":\"TOP_RIGHT_BOXAD\"",
+                "\"src\":\"gpt\""
+            )
+        },
+        {
+            "runescape",
+            "Grew",
+            "wka.gaming/_runescape//article",
+            "TOP_LEADERBOARD",
+            Arrays.asList(
+                "\"s0\":\"gaming\"",
+                "\"s0v\":\"games\"",
+                "\"s0c\":[\"gaming\"]",
+                "\"s1\":\"_runescape\"",
+                "\"s2\":\"article\"",
+                "\"ar\":\"4:3\"",
+                "\"hostpre\":\"runescape\"",
+                "\"artid\":\"33150\"",
+                "\"wpage\":\"grew\"",
+                "\"ref\":\"direct\"",
+                "\"lang\":\"en\"",
+                "\"pv\":\"1\"",
+                "\"top\":\"1k\"",
+                "\"age\":[\"kids\",\"teen\",\"under18\",\"18-24\"]",
+                "\"gnre\":[\"mmo\",\"rpg\",\"action\",\"adventure\",\"free2play\",\"fantasy\",\"mmo\",\"rpg\",\"mmorpg\",\"openworld\"]",
+                "\"pform\":[\"pc\"]",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\"pform\":[\"pc\"]" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)